apps/webapp/app/components/runs/v3/TaskRunsTable.tsx

```diff
@@ -106,173 +106,171 @@ export function TaskRunsTable({
   return (
     <Table className="max-h-full overflow-y-auto">
       <TableHeader>
-        <TableRow>
```

Comment:
"You must place `<tr>` elements inside `<thead>` to contain your `<th>` cells. The correct structure is: `<thead><tr><th>Header Cell</th></tr></thead>`. [1](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/thead) [2](https://www.w3.org/TR/html401/struct/tables.html)"